netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -340,7 +340,7 @@ private void createStream(CreateStreamCommand command, final ChannelPromise prom
       if (!connection().goAwaySent()) {
         logger.fine("Stream IDs have been exhausted for this connection. "
                 + "Initiating graceful shutdown of the connection.");
-        super.close(ctx(), ctx().newPromise());
+        close(ctx(), ctx().newPromise());
```

Comment:
For reference: https://github.com/grpc/grpc-java/commit/d192f1a6cbd20db519f0ec4e267de3cd01b5f65a#diff-759e858891647c2c91489ed45d0b2c5dL279